analysis_options.yaml

```diff
@@ -2,6 +2,10 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+analyzer:
```

Comment:
You don't need the analyzer experiment if your SDK constraints are >=2.12 , though I've noticed you need to restart any IDE/editors when that changes

Reply:
Thanks, this was left over from when I did the bulk of this PR (against 2.11.something)